sdk/cosmos/azure-cosmos/src/samples/java/com/azure/cosmos/ChangeFeedProcessorLatestVersionChangesCodeSnippet.java

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos;
+
+import com.azure.cosmos.implementation.TestConfigurations;
+import com.azure.cosmos.models.ChangeFeedProcessorItem;
+
+public class ChangeFeedProcessorLatestVersionChangesCodeSnippet {
+    public void changeFeedProcessorBuilderCodeSnippet() {
+        String hostName = "test-host-name";
+        CosmosAsyncClient cosmosAsyncClient = new CosmosClientBuilder()
+            .endpoint(TestConfigurations.HOST)
+            .key(TestConfigurations.MASTER_KEY)
+            .contentResponseOnWriteEnabled(true)
+            .consistencyLevel(ConsistencyLevel.SESSION)
+            .buildAsyncClient();
+        CosmosAsyncDatabase cosmosAsyncDatabase = cosmosAsyncClient.getDatabase("testDb");
+        CosmosAsyncContainer feedContainer = cosmosAsyncDatabase.getContainer("feedContainer");
+        CosmosAsyncContainer leaseContainer = cosmosAsyncDatabase.getContainer("leaseContainer");
+        // BEGIN: com.azure.cosmos.latestVersionChanges.builder
+        ChangeFeedProcessor changeFeedProcessor = new ChangeFeedProcessorBuilder()
+            .hostName(hostName)
+            .feedContainer(feedContainer)
+            .leaseContainer(leaseContainer)
+            .handleLatestVersionChanges(changeFeedProcessorItems -> {
+                for (ChangeFeedProcessorItem item : changeFeedProcessorItems) {
+                    // Implementation for handling and processing of each change feed item goes here
+                }
+            })
+            .buildChangeFeedProcessor();
+        // END: com.azure.cosmos.latestVersionChanges.builder
+    }
+
+    public void handleChangesCodeSnippet() {
```

Comment:
```suggestion
    public void handleLatestVersionChangesCodeSnippet() {
```

Reply:
Also, I noticed that the code snippet file I added also has the same API with same name - `handleChangesCodeSnippet`. Can you please update that as well to `handleAllVersionsAndDeletesChangesCodeSnippet()`, thanks!

Reply:
updated